app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    [Person(pers["name"], pers["age"]) for pers in people]
```

Comment:
We don't do list comprehensions in this way.
Note this requirement: Write function create_person_list, this function takes list people and **return list with Person instances** instead of dicts.
So you can assign this comprehension to variable and remove it at the end instead of generating a new list comprehension